jetcd/pom.xml

```diff
@@ -21,12 +21,8 @@
 
   <dependencies>
     <dependency>
-<!--
       <groupId>io.etcd</groupId>
       <artifactId>jetcd-core</artifactId>
- -->
-      <groupId>io.etcd</groupId>
-      <artifactId>jetcd-all</artifactId>
```

Comment:
are you 100% sure that this is right? I have a (vague) memory that I needed this so that it works in karaf (may be https://github.com/etcd-io/jetcd/issues/434 or https://github.com/etcd-io/jetcd/issues/382, can't remember), and given that you wrote that you're having trouble getting karaf to start, I'd like to have a closer look before merging this